conftest.py

```diff
@@ -1,4 +1,5 @@
 from pathlib import Path
+from unittest.mock import AsyncMock
```

Comment:
This is the only change to this file, and `AsyncMock` isn't used anywhere else in this PR. Should it remain?